bikeshed/datablocks.py

```diff
@@ -157,7 +156,7 @@ def transformDataBlocks(doc, lines):
     return newLines
 
 
-def cleanPrefix(lines, tabSize):
```

Comment:
Not sure if the `tabSize` concept got lost somewhere, since the function seems to indicate it might be relavant

Reply:
Nah it's a leftover of the previous iteration of this code: https://github.com/tabatkins/bikeshed/commit/4ef7b2950cbfcdacc8f01e63a484b17cd52ba446